rackcity/views/it_instance_views.py

```diff
@@ -24,5 +53,34 @@ def instance_detail(request, pk):
         return HttpResponse(status=404)
```

Comment:
Is this endpoint considered done or WIP? If it's done I have a couple comments

Reply:
WIP